storage/client/queue/src/main/java/com/azure/storage/common/policy/RequestRetryOptions.java

```diff
@@ -64,6 +64,11 @@ public RequestRetryOptions() {
      * <p><strong>Sample Code</strong></p>
      *
      * <p>For more samples, please see the <a href="https://github.com/Azure/azure-storage-java/blob/master/src/test/java/com/microsoft/azure/storage/Samples.java">samples file</a></p>
+     * @throws IllegalArgumentException If one of the following case exists:
+     *                                  <ul>
+     *                                      <li> There is only one null value for retryDelay and maxRetryDelay.<li/>
+     *                                      <li> Unrecognized retry policy type.<li/>
+     *                                  </ul>
```

Comment:
This is better - but the indentation is unnecessary.

Reply:
I aligned with the start point of the description. Follow the format above. Could you explain it more where these line should be?